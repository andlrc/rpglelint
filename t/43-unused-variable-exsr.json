[
   {
      "column":7,
      "file":"t/43-unused-variable-exsr.rpgle",
      "line":4,
      "message":"'my_var2' defined but not used",
      "severity":"warning"
   }
]
