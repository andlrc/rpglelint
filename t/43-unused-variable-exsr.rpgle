// rpglelint: -Wunused-variable

dcl-s my_var  int(10);
dcl-s my_var2 int(10);

exsr a;

begsr a;
  my_var = 1;
endsr;
