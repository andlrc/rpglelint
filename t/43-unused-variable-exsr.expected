t/43-unused-variable-exsr.rpgle:4:7: warning: 'my_var2' defined but not used [-Wunused-variable]
 dcl-s my_var2 int(10);
       ^
